devices/Ds1302/RtcBase.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
why don't you leave this file in the other binding? and just reference it in the nfproj? That *should* work

Reply:
Good point, I forgot to revert this...

Reply:
Reopening this comment as the file is back :-)

Reply:
If you can't get rid of it, can you please add a comment in the file, right after the header that the original one is in the Rtc folder?